src/containers/ArticlePage/FrontpageArticlePage/components/FrontpageForm.tsx

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Formik, FormikProps } from 'formik';
+import { IArticle, IUpdatedArticle, IStatus } from '@ndla/types-draft-api';
+import { AlertModalWrapper } from '../../../FormikForm';
+import validateFormik, { getWarnings } from '../../../../components/formikValidationSchema';
+import { frontPageArticleRules, isFormikFormDirty } from '../../../../util/formHelper';
+import { toEditArticle } from '../../../../util/routeHelpers';
+import HeaderWithLanguage from '../../../../components/HeaderWithLanguage';
+import EditorFooter from '../../../../components/SlateEditor/EditorFooter';
+import {
+  FrontpageArticleFormType,
+  useArticleFormHooks,
+} from '../../../FormikForm/articleFormHooks';
+import usePreventWindowUnload from '../../../FormikForm/preventWindowUnloadHook';
+import { useLicenses, useDraftStatusStateMachine } from '../../../../modules/draft/draftQueries';
+import { validateDraft } from '../../../../modules/draft/draftApi';
+import {
+  draftApiTypeToFrontpageArticleFormType,
+  frontpageArticleFormTypeToDraftApiType,
+  getExpirationDate,
+} from '../../articleTransformers';
+import { blockContentToHTML } from '../../../../util/articleContentConverter';
+import { DraftStatusType } from '../../../../interfaces';
+import StyledForm from '../../../../components/StyledFormComponents';
+import FrontPagePanels from './FrontpagePanels';
+
+interface Props {
+  article?: IArticle;
+  articleStatus?: IStatus;
+  isNewlyCreated: boolean;
+  articleChanged: boolean;
+  updateArticle: (updatedArticle: IUpdatedArticle) => Promise<IArticle>;
+  updateArticleAndStatus: (input: {
+    updatedArticle: IUpdatedArticle;
+    newStatus: DraftStatusType;
+    dirty: boolean;
+  }) => Promise<IArticle>;
+  articleLanguage: string;
+}
+
+const FrontPageForm = ({
```

Comment:
Burde uansett heite det samme som fila.
```suggestion
const FrontpageArticleForm = ({
```